net.lutris.Lutris.yml

```diff
@@ -178,6 +178,8 @@ modules:
     config-opts: *v4l2_config_opts
     sources: *v4l2_sources
 
+  - shared-modules/libusb/libusb.json
```

Comment:
we are not passing the i686 config opts to the shared module

Reply:
I only tried this locally but adding a module

```yaml
  - name: libusb-32bit
    config-opts:
      - --disable-static
    sources:
      - type: archive
        url: https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
        sha256: ffaa41d741a8a3bee244ac8e54a72ea05bf2879663c098c82fc5757853441575
    post-install:
      - install -Dm644 COPYING ${FLATPAK_DEST}/share/licenses/libusb/COPYING
    build-options:
      arch:
        x86_64: *compat_i386_opts\
```

did not help.